svn_extractor.py

```diff
@@ -8,13 +8,18 @@
 import traceback
 import re
 
+def getext(filename):
+    name, ext = os.path.splitext(filename)
+    return ext[1:]
```

Comment:
my bad in that case. I was just looking at diff without actually running it. Give me a days time i will get this tested locally and will merge. 

Based on looks everything seems to be in place.